blueprints/airflow_api.py

```diff
@@ -121,6 +121,29 @@ def dags_index():
     return ApiResponse.success({'dags': dags})
 
 
+@csrf.exempt
+@airflow_api_blueprint.route('/dags/<string:dag_id>/paused/<string:paused>',
```

Comment:
Thinking about it, we may want to change the route to be '/dags/<string:dag_id>', have the call expect a POST'ed JSON object that currently only expect { "paused": true/false }, and then change the method from POST to PATCH.  That way we can extend this later for any other update operations on a DAG.  Maybe change the function def name to be "update_dag".

Reply:
What about a `POST` to `/dags` with all the params in the object, passing in the following?
`{ "dag_id":"dag1", "paused": true/false }`

Then you could bulk update in one request:
`{ "dag_id":"dag1,dag2,dag3", "paused": true/false }`

Reply:
I think in general for the case of multiple records being updated in the same call we can add a secondary URL. ie, have both /dags and /dags/[dag_id].  However looking at the schema, since pretty much the only property of a dag that should be manually changed is the "is_paused" property, this should be fine.  However let's make the JSON be:
`{ "dag_id": ["dag1","dag2","dag3"], "paused": true/false }`, and make it a `PUT`.  Try to keep things RESTful, reserve `POST` for just creation of records.

Reply:
@cwurtz Got it. 

A thought though: Should you be able to update multiple records with varying values? So set `dag1` and `dag2` to `true` but `dag3` to `false`?

In that case, the above wouldn't work.

Reply:
If you think there is a use case for it. I'd imagine the vast majority of the time if someone is updating multiple dags, it would be to set them all as true or all as false.  Worst case they have to make two API calls. 

If you disagree and think it would be more common, we could do something like:
```
[
  { "dag_id": "dag1", "paused": true},
  { "dag_id": "dag2", "paused": false }
]
```
And optionally allow just `{ "dag_id": "dag1", "paused": false }` for updating a single dag (check the input at the start of the call and coerce it into an array to keep the logic consistent)

Reply:
I think that might be a bit much for now. I'll update the PR to include `PUT` calls and to accept multiple dag_ids but that's where I'll leave it.